### PR TITLE
Deployment Makefile

### PR DESCRIPTION
**Pull request by stevekuznetsov**
fyi @kargakis @smarterclayton @csrwng

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+.PHONY: all
+
+jenkins:
+.PHONY: jenkins
+
+prow: prow-crd prow-config prow-images hook plank jenkins-operator deck horologium splice sinker commenter
+.PHONY: prow
+
+prow-crd:
+	oc apply -f cluster/ci/config/prow/prow_crd.yaml
```
Comment (kargakis) [inline Makefile]: This shouldn't be updated often enough to justify a make target I think.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+prow-config:
+	oc apply -f cluster/ci/config/prow/config.yaml cluster/ci/config/prow/plugins.yaml
+.PHONY: prow-config
+
+prow-images:
+	oc process -f cluster/ci/config/prow/prow_images.yaml | oc apply -f -
+.PHONY: prow-images
+
+hook:
+	# secrets
```
Comment (kargakis) [inline Makefile]: Let's create/update the secrets in a separate target?

Comment (stevekuznetsov) [reply Makefile]: ack

File: Makefile (hunk 1 above)
Comment (stevekuznetsov) [reply Makefile]: nest it under something else? we want one `make` target for initial bringup at the top level

Comment (kargakis) [reply Makefile]: Ok, then this is fine.

**Comment by stevekuznetsov** [comment]
TODO: split out secrets, etc

**Pull request merged**